packages/vinxi/lib/build.js

```diff
@@ -347,14 +352,17 @@ async function createRouterBuild(app, router) {
 		});
 		await createViteBuild({
 			app: app,
+			router: router,
+			root: router.root,
 			build: {
 				ssr: true,
 				ssrManifest: true,
 				rollupOptions: {
-					input: { handler: router.handler },
+					input: { handler: join(router.root, router.handler) },
 				},
 				target: "esnext",
 				outDir: join(router.outDir + "_entry"),
+				emptyOutDir: true,
 			},
```

Comment:
Should be safe to always delete even when outDir is outisde of project root.
To be very safe it could only be set to true if outDir is inside app root.

Reply:
No its fine I like being strict about this. Causes too many stupid issues without this. Actually I manually delete all the folders before hand also but this fine too.